mlflow/store/artifact/utils/models.py

```diff
@@ -63,28 +57,31 @@ def _parse_model_uri(uri):
     if parsed.scheme != "models":
         raise MlflowException(_improper_model_uri_msg(uri))
     path = parsed.path
-    m = _MODEL_URI_REGEX.match(path)
-    if m is None:
+    if not path.startswith("/") or len(path) <= 1:
```

Comment:
is this only reachable if only the scheme is passed in?

Reply:
if you just pass `_parse_models_uri("models")`, what happens?

Reply:
Looks like it throws an exception

Reply:
Amazing